replace hardcoded gcc by $CC so it can be built with intel toolchains
Pablo Escobar - sciCORE

diff -ru mrsfast-2.6.0.4.orig/Makefile mrsfast-2.6.0.4/Makefile
--- mrsfast-2.6.0.4.orig/Makefile	2013-07-24 00:53:41.000000000 +0200
+++ mrsfast-2.6.0.4/Makefile	2014-05-07 11:35:36.394689000 +0200
@@ -1,3 +1,5 @@
+CC?=gcc
+
 ALL: mrsfast
 
 LDFLAGS=-s -static
@@ -5,5 +7,5 @@
 CFLAGS= -O2 -s 
 
 mrsfast: baseFAST.o MrsFAST.o Common.o CommandLineParser.o RefGenome.o HashTable.o  Reads.o Output.o
-	gcc $^ -o $@ ${LDFLAGS} ${LIBS}
+	$(CC) $^ -o $@ ${LDFLAGS} ${LIBS}
 	rm *.o
Only in mrsfast-2.6.0.4: mrsfast
